src/codegen/enums.rs

```diff
@@ -164,9 +164,9 @@ fn generate_enum(
         version_condition(w, env, enum_.version, false, 0)?;
         writeln!(
             w,
-            "impl fmt::Display for {} {{\n\
+            "impl fmt::Display for {0} {{\n\
              \tfn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {{\n\
-             \t\twrite!(f, \"Self::{{}}\", match *self {{",
+             \t\twrite!(f, \"{0}::{{}}\", match *self {{",
```

Comment:
Strange, why does this not just do `<Self as fmt::Debug>::fmt(self, f)` like for flags? But yeah, I don't agree with the existence of this `Display` impl anyway ;)

Reply:
It's optional and seemingly only enabled for gtk(4)-rs which is why I didn't notice :)

Not that the `#[derive(Debug)]` is all too useful, tried this on `GLSLVersion` and it only prints the variant name (like `_430`), excluding the type.

Reply:
In that sense this could be a `write!(f, "{}::{:?}", stringify(#typename), self)`?